lib/rx/configuration.rb

```diff
@@ -59,8 +59,8 @@ def connection
         conn.request :json
 
         # Uncomment this if you want curl command equivalent or response output to log
-        # conn.request(:curl, ::Logger.new(STDOUT), :warn) unless Rails.env.production?
-        # conn.response(:logger, ::Logger.new(STDOUT), bodies: true) unless Rails.env.production?
+        conn.request(:curl, ::Logger.new($stdout), :warn) unless Rails.env.production?
+        conn.response(:logger, ::Logger.new($stdout), bodies: true) unless Rails.env.production?
```

Comment:
Change this env to staging to test before we try prod

Reply:
that uses an `unless` so should log in any env that isn't production.

Reply:
lol whoops